c2corg_ui/static/js/preferences.js

```diff
@@ -0,0 +1,157 @@
+goog.provide('app.PreferencesController');
+goog.provide('app.preferencesDirective');
+
+goog.require('app');
+goog.require('app.Alerts');
+goog.require('app.Api');
+goog.require('app.Authentication');
+goog.require('app.utils');
+
+
+/**
+ * Directive managing the user preferences.
+ * @return {angular.Directive} The directive specs.
+ * @ngInject
+ */
+app.preferencesDirective = function() {
+  return {
+    restrict: 'A',
+    scope: true,
+    controller: 'appPreferencesController',
+    controllerAs: 'prefCtrl',
+    bindToController: true
```

Comment:
if you write `scope: true` and `bindToController: true`, what actually is bound to the controller when you don't set any value for example with `{someObject: '='}` ? is it of any use in this case?